src/PrivateDns/PrivateDns/help/Remove-AzPrivateDnsVirtualNetworkLink.md

```diff
@@ -0,0 +1,231 @@
+You can pass a **PSPrivateDnsVirtualNetworkLink** object using the *Link* parameter or by using the pipeline operator, or alternatively you can specify the *Name* *ZoneName* and *ResourceGroupName* parameters.
+You can use the Confirm parameter and $ConfirmPreference Windows PowerShell variable to control whether the cmdlet prompts you for confirmation.
+When specifying the link using a **PSPrivateDnsVirtualNetworkLink** object (passed via the pipeline or *Link* parameter), the link is not deleted if it has been changed in Azure Private DNS since the local **PSPrivateDnsVirtualNetworkLink** object was retrieved. This provides protection for concurrent zone changes. This can be suppressed using the *Overwrite* parameter, which deletes the zone regardless of concurrent changes.
+
+## EXAMPLES
+
+### Example 1: Remove a link
+```
+PS C:\>Remove-AzPrivateDnsVirtualNetworkLink -ResourceGroupName "MyResourceGroup" -ZoneName "myzone.com" -Name "mylink"
+```
+
+This command removes the link named mylink linked to zone myzone.com from the resource group named MyResourceGroup.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+The virtual network link object to remove.
+
+```yaml
+Type: Microsoft.Azure.Commands.PrivateDns.Models.PSPrivateDnsVirtualNetworkLink
+Parameter Sets: Object
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Name
+Specifies the name of the link to be deleted.
+You must also specify the *ResourceGroupName* and *ZoneName* parameter.
+Alternatively, you can specify the link using the *Link* parameter.
+
+```yaml
+Type: System.String
+Parameter Sets: Fields
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Overwrite
+When specifying the zone using a **PSPrivateDnsVirtualNetworkLink** object (passed via the pipeline or *Link* parameter), the zone is not deleted if it has been changed in Azure DNS since the local **PSPrivateDnsVirtualNetworkLink** object was retrieved.
+This provides protection for concurrent zone changes.
+This can be suppressed using the *Overwrite* parameter, which deletes the zone regardless of concurrent changes.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: Object
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassThru
+Used for passing the deleted private zone further down the pipeline.
```

Comment:
I don't think this is true - when PassThru is used, bool is output to indicate success, not the deleted private zone. Please update.

Reply:
yeah you are right, i just checked it.